app/web/features/messages/messagelist/InfiniteMessageLoader.tsx

```diff
@@ -74,7 +74,7 @@ export default function InfiniteMessageLoader({
     const messageEl = document.getElementById(
       messageElementId(prevTopMessageId.current ?? 0)
     );
-    messageEl?.scrollIntoView();
+    messageEl?.scrollIntoView({ block: "center" });
```

Comment:
This is still not perfect, but I find that without this tweak, when the user scroll up, there's a weird scroll jerk where the whole page gets scrolled. Scrolling into view to align to the center of the screen at least seems to have reduce the sudden jerky movement